package.json

```diff
@@ -49,28 +49,64 @@
   },
   "scripts": {
     "bootstrap:hoist": "lerna bootstrap --hoist --force-local",
-    "clean": "lerna clean --yes && rm -rf node_modules || true",
     "clean:install": "rm package-lock.json; npm run clean && npm install",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
+    "compile:prod": "rm -rf ./build; NODE_ENV=production webpack -p --config webpack.dev-site.config.js",
     "compile": "lerna run compile --parallel",
     "danger": "danger ci",
+    "heroku-postbuild": "npm install --only=dev && npm run compile:prod",
+    "heroku-prebuild": "npx lerna init",
     "jest": "jest",
-    "mocha": "cd packages/terra-functional-testing && npm run mocha",
-    "lint": "npm run lint:js",
     "lint:js": "eslint --ext .js,.jsx .",
+    "lint": "npm run lint:js",
+    "mocha": "cd packages/terra-functional-testing && npm run mocha",
     "pack": "webpack -p",
     "postinstall": "link-parent-bin && npm run bootstrap:hoist",
+    "start": "webpack-dev-server --config webpack.dev-site.config.js",
+    "start-heroku": "node ./scripts/hack-express-server.js --port $PORT --site './build' ",
     "test": "npm run lint && npm run jest && npm run mocha && npm run wdio",
-    "wdio": "rm -rf build && npm run pack && npm run wdio-site && npm run wdio-webpack",
+    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js",
     "wdio-site": "SITE=build node ./packages/terra-functional-testing/bin/test-runner.js",
     "wdio-webpack": "node ./packages/terra-functional-testing/bin/test-runner.js",
-    "version": "lerna version --no-git-tag-version && node ./scripts/changelog-updater/index.js"
+    "wdio": "rm -rf build && npm run pack && npm run wdio-site && npm run wdio-webpack"
   },
   "devDependencies": {
+    "@babel/cli": "^7.10.5",
+    "@babel/core": "^7.11.1",
+    "@babel/helpers": "^7.10.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
+    "@babel/plugin-syntax-async-generators": "^7.8.4",
+    "@babel/plugin-transform-regenerator": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.11.0",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.10.5",
+    "@wdio/cli": "^6.3.6",
+    "@wdio/local-runner": "^6.3.6",
+    "@wdio/mocha-framework": "^6.3.6",
+    "@wdio/selenium-standalone-service": "^6.1.14",
+    "@wdio/sync": "^6.3.6",
+    "browserslist-config-terra": "^1.2.0",
```

Comment:
Yeah, this is weird, I had to add this to get the webpack build to succeed. I think this is related to how updating this in our packages isn't working as expected.

Reply:
Does the root package.json need to [extend a browserslist](https://github.com/cerner/terra-core/blob/main/package.json#L24-L26)? If it does then we might also run into [this issue](https://github.com/browserslist/browserslist/issues/484).

Reply:
I duplicated the config to the package.json, it doesn't really matter for this repo tbh.